README.md

```diff
@@ -1 +1,166 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Game Statistics] <a name="about-project"></a>
+
+**[Game Statistics]** "Game Statistics" is a website for all fans of American Football Games. It is a Single Page App (SPA) that allows users to view Game statics for a given season.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="#">React.js</a></li>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Home page with game Metrix]**
+- **[Home page with search games field]**
+- **[Details page component]**
+- **[Back Button to the home page]**
+- **[Showing pages using React Router]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+Have a laptop,
+Install Git,
+Install Node.js,
+Have a working browser,
+Have a strong internet connection,
+Install VSCode with with extentions like live server, prettier formatter etc.
+
+### Setup
+
+Install [Node Js](https://nodejs.org/en) and [NPM](https://docs.npmjs.com/cli/v6/commands/npm-install) and [Git](https://git-scm.com/downloads) on your local computer.
+
+### Install
+
+Open your commandline (if your are on windows ) or Terminal (if you are on Mac or Linux) and paste this command:
+
+```sh
+git clone https://github.com/enocol/Metrics-webapp
+```
+
+then you will see the **Metrics-webapp** on your system then type this:
+
+```sh
+cd Metrics-webap
+```
+
+Now you successfully installed it.
+
+### Usage
+
+Inside **Metrics-webap** on a terminal, type npm i to install dependencies
+After dependencies are installed, type npm start
+
+### Run tests
+
+To run tests, run the following command:
+in a ternimal, type npm run test
+
+### Deployment
+
+You can deploy this project using:
+
+**Github pages**
+**Renda**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Enoh Collins</a>
+
+👤 **Enoh Collins**
+
+- GitHub: [@githubhandle](https://github.com/enocol)
+- Twitter: [@twitterhandle](https://mobile.twitter.com/enocol)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/enocol)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Contact component]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+live Demo link <h5><a href="https://metrics-ball.onrender.com/">Live Demo</a></h5>
+Loom Video Link <h5>a href="https://www.loom.com/share/a4bfdc5fc3fb479c80317071adc9f6b4"/>Loom Video</a></h5>
```

Comment:
- [ ] Hey there! 😄 As previously requested, it would be awesome if you could create a video showing an interesting piece of code that you've written. 🎥🚀 Sharing your code visually can be super helpful and engaging!